OsmAnd/src/net/osmand/plus/AppInitializer.java

```diff
@@ -194,6 +194,10 @@ public void initVariables() {
 				app.getSettings().migrateGlobalPrefsToProfile();
 				startPrefs.edit().putInt(VERSION_INSTALLED_NUMBER, VERSION_3_5).commit();
 			}
+			if (prevAppVersion < VERSION_3_5 || Version.getAppVersion(app).equals("3.5.3")) {
```

Comment:
Why duplicated if?

Reply:
It's for migration when Fix will be applied version number will be different